contracts/IDrop.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
```

Comment:
Can we use the latest LTS solidity version?

Reply:
![image](https://user-images.githubusercontent.com/29498872/147852285-8a511332-4aac-4e02-b89e-e17a26ae840b.png)

Reply:
The current solidity version is 0.8.11.

Hardhat supports upto 0.8.9. Most contracts use 0.8.4. Do we have any specific reason to upgrade to 0.8.9?

Reply:
Yes 0.8.4 has security vulnerabilities, try using within 0.8.9 to 0.8.11

The above screenshot is for hardhat support!

Reply:
<img width="336" alt="Screenshot 2022-01-01 at 7 36 02 PM" src="https://user-images.githubusercontent.com/69431456/147852382-2c87da19-fdb5-4889-b8d9-1346f413345a.png">

Reply:
We can use 0.8.9 then!

Reply:
Updated to 0.8.9